### PR TITLE
Updates for v0.6

### PR DESCRIPTION
**Pull request by paulo-valim**
Added at the features session: Smart Cells
PS: the image that's being used is temporary.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -186,6 +186,29 @@ <h3 class="text-xl text-gray-50 font-semibold">
         </div>
       </div>
     </div>
+    <!--Smart Cells-->
+    <div
+      class="flex flex-col md:flex-row items-center space-y-8 md:space-y-0 md:space-x-8 lg:space-x-28"
+    >
+      <div class="flex flex-col align-ite mt-8">
+        <h3 class="text-xl text-gray-50 font-semibold">
+          &#9889; Smart cells
```
Comment (jonatanklosko) [inline src/index.html:195]:
```suggestion
          Smart cells
```

@josevalim please check the preview and let me know if you still want the icon. Alternatively we can use the remix icon, however I think we are better off without the icon, for consistency :)

File: src/index.html [hunk 2]
```diff
@@ -197,7 +220,7 @@ <h3 class="text-xl text-gray-50 font-semibold">
         width="448"
         height="162"
       />
-      <div class="flex flex-col">
+      <div class="flex flex-col align-ite">
```
Comment (jonatanklosko) [inline src/index.html:223]: :cat:

```suggestion
      <div class="flex flex-col">
```

File: src/index.html (hunk 1 above)
Comment (josevalim) [reply src/index.html:202]: Anyway is fine. It is your call!

Comment (jonatanklosko) [reply src/index.html:202]: In that case let's drop it :D

File: src/index.html [hunk 3]
```diff
@@ -186,6 +186,29 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+        width="448"
+        height="162"
+      />
+      <div class="flex flex-col md:mr-8 lg:mr-28">
+        <h3 class="text-xl text-gray-50 font-semibold">
+          &#9889; Smart cells
+        </h3>
+        <span class="mt-3 md:max-w-md text-gray-300">
+          Use UI components to master new workflows and
+          write code faster than ever. Join our growing
```
Comment (josevalim) [inline src/index.html:206]:
```suggestion
          write notebooks faster than ever. Join our growing
```

File: src/index.html [hunk 4]
```diff
@@ -186,6 +186,29 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+      <div class="flex flex-col md:mr-8 lg:mr-28">
+        <h3 class="text-xl text-gray-50 font-semibold">
+          Smart cells
+        </h3>
+        <span class="mt-3 md:max-w-md text-gray-300">
+          Use UI components to master new workflows and
+          write notebooks faster than ever. Join our growing
+          community to automate more, from HTTP requests
+          to machine learning models.
+        </span>
```
Comment (josevalim) [inline src/index.html:209]:
```suggestion
        <h3 class="text-xl text-gray-50 font-semibold">
          Automate workflows
        </h3>
        <span class="mt-3 md:max-w-md text-gray-300">
          Use Smart Cells to perform high-level tasks and
          write notebooks faster than ever. Join our growing
          community to automate more, from HTTP requests
          to Machine Learning models.
        </span>
```

File: src/index.html [hunk 5]
```diff
@@ -186,6 +186,29 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+        height="162"
+      />
+      <div class="flex flex-col md:mr-8 lg:mr-28">
+        <h3 class="text-xl text-gray-50 font-semibold">
+          Automate workflows
+        </h3>
+        <span class="mt-3 md:max-w-md text-gray-300">
+          Use Smart Cells to perform high-level tasks and
+          write notebooks faster than ever. Join our growing
+          community to automate more, from HTTP requests
```
Comment (josevalim) [inline src/index.html:207]:
```suggestion
          community to automate more, from Database queries
```

File: src/index.html [hunk 6]
```diff
@@ -177,6 +177,26 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+        alt=""
+        width="550"
+        height="280"
+      />
+      <div class="flex flex-col md:mr-8 lg:mr-28">
+        <h3 class="text-xl text-gray-50 font-semibold">Automate workflows</h3>
+        <span class="mt-3 md:max-w-md text-gray-300">
+          Use Smart Cells to perform high-level tasks and write notebooks faster
+          than ever. Join our growing community to automate more, from database
+          queries to Machine Learning models.
```
Comment (josevalim) [inline src/index.html:196]:
```suggestion
          Use Smart cells to perform high-level tasks and write notebooks faster
          than ever. ⚡ Join our growing community to learn and master new workflows,
          from database queries to Machine Learning models.
```

File: src/index.html [hunk 7]
```diff
@@ -177,6 +177,26 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+        src="/images/smart_cells.png"
+        alt=""
+        width="550"
+        height="280"
+      />
+      <div class="flex flex-col md:mr-8 lg:mr-28">
+        <h3 class="text-xl text-gray-50 font-semibold">Automate workflows</h3>
+        <span class="mt-3 md:max-w-md text-gray-300">
+          Use Smart cells to perform high-level tasks and write notebooks faster
+          than ever. ⚡ Join our growing community to learn and master new workflows,
```
Comment (josevalim) [inline src/index.html:195]:
```suggestion
          than ever! Join our growing community to learn and master new workflows,
```

**Review by josevalim — APPROVED** [review]

File: src/index.html [hunk 8]
```diff
@@ -193,9 +213,9 @@ <h3 class="text-xl text-gray-50 font-semibold">
           Write once, read anywhere
         </h3>
         <span class="mt-3 md:max-w-md text-gray-300">
-          Notebooks are stored as .livemd files, a subset of Markdown
-          that meshes together prose, code, diagrams, and more. You can
-          easily share, read, and version them within your existing tooling.
+          Notebooks are stored as .livemd files, a subset of Markdown that
+          meshes together prose, code, diagrams, and more. You can easily share,
+          read, and version them within your existing tooling.
```
Comment (josevalim) [inline src/index.html:218]:
```suggestion
          Notebooks are stored as .livemd files, a subset of Markdown that
          meshes together prose, code, and diagrams. Livebooks are easy
          to read, share, and version within your existing tooling.
```

File: src/index.html [hunk 9]
```diff
@@ -122,6 +122,26 @@ <h3 class="text-xl text-gray-50 font-semibold">Interactive data</h3>
...
+        <h3 class="text-xl text-gray-50 font-semibold">Automate workflows</h3>
+        <span class="mt-3 md:max-w-md text-gray-300">
+          Use Smart cells to perform high-level tasks and write notebooks faster
+          than ever! Join our growing community to learn and master new workflows,
+          from database queries to Machine Learning models.
+        </span>
+      </div>
+    </div>
     <!-- Screenshot with features -->
     <div class="lg:py-16 max-w-7xl">
```
Comment (josevalim) [inline src/index.html:146]:
```suggestion
    <div class="lg:py-8 max-w-7xl">
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**